### PR TITLE
Ordre des champs + titres en gras

### PR DESCRIPTION
**Pull request by celiacheff**
[Trello 1](https://trello.com/c/3nQtzRJi/697-ordre-des-informations-sur-laccueil)
[Trello 2](https://trello.com/c/Ct4vVuMU/696-passer-en-gras-les-titres-dans-la-fiche-erp)

### CODE REVIEW

File: templates/erp/includes/accessibilite.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
   {% if info.fields %}
     <h2 class="h5 mb-0 pb-1 border-bottom">
       <i aria-hidden="true" class="icon icon-{{ info.icon }} mr-1"></i>
-      {{ section }}
+      <b>{{ section }}</b>
```
Comment (n1k0) [inline templates/erp/includes/accessibilite.html:14]: This is semantically incorrect, please use a `strong` tag

File: templates/erp/includes/accessibilite.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
   {% if info.fields %}
     <h2 class="h5 mb-0 pb-1 border-bottom">
       <i aria-hidden="true" class="icon icon-{{ info.icon }} mr-1"></i>
-      <b>{{ section }}</b>
+      <span class="font-weight-bold">{{ section }}</span>
```
Comment (n1k0) [inline templates/erp/includes/accessibilite.html:14]: You're gonna hate me but the class should rather be added to the parent `h3` tag.

Comment (celiacheff) [reply templates/erp/includes/accessibilite.html]: No problems :) the task wasn't described formally anyways, so I just emphasized the already visible part. Tell me if this if better

**Review by n1k0 — APPROVED** [review]
Looks good, I pushed a commit to make the vaccination section heading bold as well.

**Pull request merged**